backend/core/src/user/dto/forgot_password.dto.ts

```diff
@@ -0,0 +1,21 @@
+import { IsEmail, IsString, IsOptional, MaxLength } from "class-validator"
+import { Expose } from "class-transformer"
+import { ValidationsGroupsEnum } from "../../shared/validations-groups.enum"
+
+export class ForgotPasswordDto {
+  @Expose()
+  @IsEmail({}, { groups: [ValidationsGroupsEnum.default] })
+  @MaxLength(256, { groups: [ValidationsGroupsEnum.default] })
+  email: string
+
+  @Expose()
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @IsString({ groups: [ValidationsGroupsEnum.default] })
+  @MaxLength(256, { groups: [ValidationsGroupsEnum.default] })
```

Comment:
@software-project Is there a chance that this will be too small for an actual URL?

Reply:
@pbn4 I don't think so. It's just the base app url.